tap_pendo/streams.py

```diff
@@ -898,6 +781,7 @@ class Reports(Stream):
     name = "reports"
     replication_method = "INCREMENTAL"
     replication_key = "lastUpdatedAt"
+    endpoint = Endpoints("/api/v1/report", "GET")
```

Comment:
Add unit test case for this code change

Reply:
Have added one common unit test case for this change in the test_endpoints_dict.py file